### DISCUSSION TITLE
Please help, Flash Builder crash with the latests AIRSDK

### DISCUSSION THREAD

**Discussion by 3x1010** [airsdk/Adobe-Runtime-Support#1463] [Q&A] [answered]
Hello there,
Maybe someone had already my issue and can help me.
I just overlayed the new harman SDK (713) in Flash Builder, I didn't do it since a while but when I open Flash Builder I get this message:

**Could not open the editor: The editor class could not be instantiated. This usually indicates a missing no-arg constructor or that the editor's class name was mistyped in plugin.xml.**

If I switch back to the AIRSDK 406 all works fine.
I tried also with the version 633 and it crashes.
Any idea how to fix this?
Thanks in advance

**Comment by 2jfw** [comment]
How exactly did you "overlay"?
(You normally only have to overlay when you also use Flex SDK)

**Reply by ajwfrost** [reply]
Yes - would be good to know what steps you took and what was there previously.
Plus, can you look for the error log from Flash Builder and see if there are any more details?

thanks

**Comment by 3x1010** [comment, marked as answer]
Sorry maybe I used the wrong word.
This is what I usually do.

I went here:
C:\Program Files\Adobe\Adobe Flash Builder 4.7 (64 Bit)\eclipse\plugins\com.adobe.flash.compiler_4.7.0.349722
Then I removed the old AIRSDK directory.
Then I copied the whole content of the airsdk I just downloaded ( https://airsdk.harman.com/api/versions/33.1.1.713/sdks/AIRSDK_Windows.zip?id=74479 ) in a new AIRSDK directory
If I run Flash Builder it gives me that error
Could not open the editor: The editor class could not be instantiated. This usually indicates a missing no-arg constructor or that the editor's class name was mistyped in plugin.xml.

I have done this operation a lot of times in the past and I never had any problem (in fact when I switched back to version 406 all worked fine)

Here it is the log, it looks like Greek to me:
[log.txt](https://github.com/airsdk/Adobe-Runtime-Support/files/7726026/log.txt)

Thanks!

**Reply by ajwfrost** [reply]
Ah - thanks, that log file has it:
```
Caused by: java.lang.UnsupportedClassVersionError: com/adobe/flash/compiler/definitions/IDefinition : Unsupported major.minor version 51.0
```

Can you please update the Java runtime that's within Flash Builder's "jre" folder? Basically grab something from Java 1.8 upwards, install it normally, and then copy it over to the "C:\Program Files\Adobe\Adobe Flash Builder 4.7 (64 Bit)\jre" folder?

thanks

**Comment by 3x1010** [comment]
Thanks a lot mate!
It solved the issue.
And it'sall my fault because when I had the licence problem with Flash Builder I surely uninstalled it and then reinstalled and I forgot to upgrade the jre...
The strange thing is that with the 406 all worked fine!
Anyway, thanks a lot!
